src/finch/julia.py

```diff
@@ -30,3 +30,4 @@
 
 jl.seval("using Finch")
 jl.seval("using Random")
+jl.seval("using TensorMarket")
```

Comment:
Will this limit us to just this format?

Reply:
No, it's just a dependency. Here's a list of all supported formats: https://willowahrens.io/Finch.jl/stable/guides/fileio/#Finch.fread

Reply:
Do we need to pull in any other dependencies for other formats?

Reply:
@willow-ahrens Are there other IO related dependencies that we should install during the setup to fully support `fread` and `fwrite`?

In `Finch.jl` in `Project.toml` I found three more:

```toml
[weakdeps]
HDF5 = "f67ccb44-e63f-5c2f-98bd-6dc0ccc4ba2f"
NPZ = "15e1cf62-19b3-5cfa-8e77-841668bca605"
SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
TensorMarket = "8b7d4fe7-0b45-4d0d-9dd8-5cc9b23b4b77"
```

Reply:
HDF5, NPZ, and TensorMarket are required for binsparse and ttx files.

Reply:
Thanks! I load them dynamically.